app/views/govuk_component/taxonomy_sidebar.raw.html.erb

```diff
@@ -4,20 +4,24 @@
     <% items.each_with_index do |item, item_index| %>
       <div class='sidebar-taxon'>
         <h2>
-          <%=
-            link_to(
-              item[:title],
-              item[:url],
-              class: 'taxon-link',
-              data: {
-                track_category: 'relatedLinkClicked',
-                track_action: "#{item_index + 1}",
-                track_label: item[:url],
-                track_dimension: item[:title],
-                track_dimension_index: 29,
-              },
-            )
-          %>
+          <% if item[:url] %>
```

Comment:
Maybe you could try http://apidock.com/rails/ActionView/Helpers/UrlHelper/link_to_if here.